examples/step-55/doc/tooltip

```diff
@@ -0,0 +1 @@
+Precondtioners: Geometric multigrid vs ILU
```

Comment:
typo.

Also, maybe rephrase this to state the actual purpose of the program: "Geometric multigrid preconditioners for the Stokes problem"